### ISSUE TITLE
Versioning and release schedule

### ISSUE THREAD

**Issue by Flyingmana** [OpenMage/organizational#9]
As we are approaching the point where we want to officially accept patches which may break some compatibility or extend the scope of bugfixes, choosing proper versioning is an important step.

we will keep the 1.9.x branches for now with the current rules for applying patches.

For the new Versions I suggest basing the version number on the year. Its far enough away from the magento release numbers to avoid confusion regarding compatibility, and does not occupy version numbers which may be used in the near future in any circumstances. (MariaDB did a similar approach after diverging from MySql)
Also starting a new release in the beginning of the year matches good with the schedule of online shops, which are mostly driven by holiday sales.

Therefore the first Version containing additional changes would be 18.x and be released end(?) of January. The branch containing first alpha releases would be created already during december.
This would repeat every year.
A version would be eol 2 years after the first stable release. 

Every few years we could mark one as LTS release which is provided with patches for a longer time, but I would wait with planning this for a while.
At least till we have a certain set of needed features and done tasks which should be done before. (Test suite, performance regression checks, default way to install and some other stuff)

During one Major release we will sometimes do minor releases, which can contain additional features or minor breaks. I have no idea yet how to decide about them.

What is your opinion about this, and do you have suggestions or change requests?

**Comment by sreichel** [comment]
Sounds good and like Very-LTS :)

Is it possible for the near future to add or split this into two (ore three) braches?

- `1.9.3.x` keep it as it as (backward compa)
  - less strict `18.3.x` (that accepts bc breaking changes)
    - and may some for cleanup (really remove possible dead/deprecated code)

2nd and 3rd are maybe same ...

**Comment by JustinElst** [comment]
sounds good!

**Comment by kkrieger85** [comment]
@Flyingmana Is this issue closed by https://github.com/OpenMage/organizational/blob/master/Roadmap/releases.md ??

**Comment by Flyingmana** [comment]
kind of, but its only what I created for now.

But we also have now https://github.com/OpenMage/rfcs/pull/2/files so this Issue is not needed anymore
